model-optimizer/mo/front/extractor.py

```diff
@@ -131,6 +131,20 @@ def attr_getter(node: Node, name: str):
     return None
 
 
+def bool_to_str(node: Node, attr: str):
+    # Function converts 0/1 or bool False/True values to str 'false'/'true' which need to appear in IR
+    attribute_name = node.soft_get(attr)
+    if attribute_name == '<UNKNOWN>':
+        return
```

Comment:
```py
attr_value = node.soft_get(attr, None)
if attr_value is None:
    return None
```

then use `attr_value` instead of node[attr]` in the rest of the code in this function

Reply:
Done.

Reply:
Why do you still use `<UNKNOWN>`? I suggested to use `None` as a default value.

Reply:
Ok, done.

Reply:
Resolved, sorry for misunderstanding.